src/stp/plays/offensive/AttackingPass.cpp

```diff
@@ -176,20 +203,20 @@ void AttackingPass::calculateInfoForPass(const world::ball::Ball* ball) noexcept
     stpInfos["passer"].setShotType(ShotType::TARGET);
 }
 
-bool AttackingPass::isValidPlayToKeep(world::World* world) noexcept {
-    world::Field field = world->getField().value();
-    auto closestToBall = world->getWorld()->getRobotClosestToBall();
-    auto canKeep = std::all_of(keepPlayInvariants.begin(), keepPlayInvariants.end(), [world, field](auto& x) { return x->checkInvariant(world->getWorld().value(), &field); }) &&
-                   !passFinished();
-    if (canKeep) {
-        if (closestToBall && closestToBall->get()->getTeam() == world::us) {
-            return true;
-        } else if (world->getWorld()->getBall().value()->getVelocity().length() > control_constants::BALL_STILL_VEL) {
-            return true;
-        }
-    }
-    return false;
-}
+/// To be reimplemented
+//bool AttackingPass::isValidPlayToStart(PlayEvaluator* playEvaluator) noexcept {
+//    auto closestToBall = world->getWorld()->getRobotClosestToBall();
+//    auto canKeep = std::all_of(keepPlayInvariants.begin(), keepPlayInvariants.end(), [world, field](auto& x) { return x->checkInvariant(world->getWorld().value(), &field); }) &&
+//                   !passFinished();
+//    if (canKeep) {
+//        if (closestToBall && closestToBall->get()->getTeam() == world::us) {
+//            return true;
+//        } else if (world->getWorld()->getBall().value()->getVelocity().length() > control_constants::BALL_STILL_VEL) {
+//            return true;
+//        }
+//    }
+//    return false;
+//}
```

Comment:
Try to avoid this, this will start to add up if you do this in multiple pr's. It's better to just have a temporary implementation and make it an issue or coordinate it in another way.

Reply:
Ill try to do it next times.